queries.sql

```diff
@@ -23,3 +23,32 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg).
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
```

Comment:
- [ ] You are missing two queries. 
1. You must select species from animals and rollback
2. After rollback you need to make sure you selected species from animals again.

Reply:
This was not part of the requirements please, check to verify

Reply:
Let's look through them one by one.

![Screen Shot 2022-10-19 at 16 55 57](https://user-images.githubusercontent.com/69011963/196684417-158b5c22-c434-4e34-b9be-d98cb52f60e9.png)